init/sysinit.go

```diff
@@ -132,6 +132,10 @@ func runContainersFrom(startFrom string, cfg *config.Config, containerConfigs []
 				log.Errorf("Failed to run %v: %v", containerConfig.Id, container.Err)
 			}
 
+			if cfg.Debug {
+				container.Log(os.Stdout, os.Stderr)
```

Comment:
I understand that you want blocking behavior to see the output of the containers sequentially, but the problem ends up being that not all containers stop.  For example userdocker container doesn't stop.  I believe what will happen is that it will just stop and wait for the user docker container to finish and the boot will hang.